File: WindowsFormsApp1/Models/Customers.cs
using System;
using System.Collections.Generic;

namespace WindowsFormsApp1.Models
{
    public partial class Customers
    {
        public Customers()
        {
            Orders = new HashSet<Orders>();
        }

        public string CustomerId { get; set; }
        public int Identifier { get; set; }
        public string CompanyName { get; set; }
        public string ContactName { get; set; }
        public string ContactTitle { get; set; }
        public int? ContactTitleId { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string Region { get; set; }
        public string PostalCode { get; set; }
        public string Country { get; set; }
        public string Phone { get; set; }
        public string Fax { get; set; }

        public virtual CustomersContactTitle ContactTitleNavigation { get; set; }
        public virtual ICollection<Orders> Orders { get; set; }
    }
}

File: Converted/Models/Customers.vb
Imports System
Imports System.Collections.Generic

Namespace WindowsFormsApp1.Models
	Partial Public Class Customers
		Public Sub New()
			Orders = New HashSet(Of Orders)()
		End Sub

		Public Property CustomerId() As String
		Public Property Identifier() As Integer
		Public Property CompanyName() As String
		Public Property ContactName() As String
		Public Property ContactTitle() As String
		Public Property ContactTitleId() As Integer?
		Public Property Address() As String
		Public Property City() As String
		Public Property Region() As String
		Public Property PostalCode() As String
		Public Property Country() As String
		Public Property Phone() As String
		Public Property Fax() As String

		Public Overridable Property ContactTitleNavigation() As CustomersContactTitle
		Public Overridable Property Orders() As ICollection(Of Orders)
	End Class
End Namespace
